src/execution/futex-emulation.cc

```diff
@@ -401,6 +415,8 @@ Object FutexEmulation::WaitSync(Isolate* isolate,
     node->wait_location_ = wait_location;
     node->waiting_ = true;
 
+    int node_rr_id = recordreplay::PointerId(node);
```

Comment:
Naming: Can we avoid arbitrary abbreviations? `record_replay_id` was what we used a lot, or in this case, `pointerId` also seems reasonable etc.

Reply:
It's just a local variable, but sure - I can rename it to `node_record_replay_id`.